FACTOR.py

```diff
@@ -654,12 +654,18 @@ def mine():
     global SCRIPTPUBKEY
     if SCRIPTPUBKEY == None:
         SCRIPTPUBKEY = sys.argv[1].strip()
+        
+    os.system("mkdir logs")
```

Comment:
Should be ``mkdir -p logs`` so that if the directory already exists, it does not give an error.

Reply:
you are right